module/VuFind/src/VuFind/Search/Base/Results.php

```diff
@@ -30,6 +30,7 @@
 use Laminas\Paginator\Paginator;
 use VuFind\Record\Loader;
 use VuFind\Search\Factory\UrlQueryHelperFactory;
+use VuFind\Search\Solr\HierarchicalFacetHelper;
```

Comment:
I'm not sure I like having the base class dependent on a Solr-specific helper. Should we define an interface to make this more generic/flexible?

Reply:
Yes, good idea. Now done.